src/Core/src/Core.csproj

```diff
@@ -21,7 +21,6 @@
   <ItemGroup Condition="$(TargetFramework.Contains('-windows'))">
     <PackageReference Include="Microsoft.WindowsAppSDK" />
     <PackageReference Include="Microsoft.Graphics.Win2D" />
-    <ProjectReference Include="..\..\Graphics\src\Graphics.Win2D\Graphics.Win2D.csproj" />
```

Comment:
Maui no longer uses the Win2D version, so this is a breaking change as we now use new types and don't automatically bring this package in.